packages/core/src/accordion/accordion-section.element.ts

```diff
@@ -80,12 +99,10 @@ export class CdsAccordionPanel extends LitElement implements Animatable {
         type="button"
         @click="${() => this.toggle()}"
         ?disabled="${this.disabled}"
-        aria-disabled="${this.disabled}"
-        aria-expanded="${this.expanded}"
```

Comment:
aria-expanded should be associated with the button element, as it was previously.